gammapy/cube/make.py

```diff
@@ -93,10 +98,11 @@ def run(self, observations, selection=None):
             maps_obs = obs_maker.run(selection)
 
             for name in selection:
-                data = maps_obs[name].quantity.to_value(maps[name].unit)
                 if name == "exposure":
+                    data = maps_obs[name].quantity.to_value(maps[name].unit)
                     maps[name].fill_by_coord(obs_maker.coords_etrue, data)
-                else:
+                elif name == "counts" or name == "background":
```

Comment:
Ok, to just omit the stacking of the `psf` and `edisp`, but maybe change the default of the `MapMaker.run()` to not compute those maps, otherwise it will slow down our current tutorials and user scripts a lot for no reason.

Reply:
Ok, applied a selection